web/css/codeworld-cm.css

```diff
@@ -164,6 +164,22 @@ div.CodeMirror span.cm-matchhighlight {
 
 /* Style overrides for the autocomplete box and doc-on-hover. */
 
+.function-tooltip-styling {
```

Comment:
Rather than copying this, can you split up the existing rule below, and add this new class to the list for the first part?